assets/test/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "swift.buildPath": "/some/other/full/test/path"
```

Comment:
Is this meant to be included in the PR?

Reply:
The file is created as a result of running tests. I figures if we are saving test assets anyway it might have sense to keep it in repository. I can remove it if you feel like it should not be in the repository

Reply:
It will effect other tests, so not sure we should have it here, maybe the build path tests should be reseting this once they are done.

Reply:
Yea, removing the actual setting is something the test teardown should perform, I'll add necessary code there. Tho I would still leave the empty settings.json, since there's also a test in WorkspaceContest.tests.ts that updates settings (tho that one is weird and I'm not sure it works correctly, since it is using `vscode.workspace(...).getConfiguration(...).update(...)` function synchronously, while it is supposed to be async)

Reply:
An empty settings.json is fine